build.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 image="docker.io/antora/antora"
-cmd="--html-url-extension-style=indexify site.yml"
+cmd="site.yml"
```

Comment:
Will this [change our URLs](https://docs.antora.org/antora/latest/playbook/urls-html-extension-style/#html-extension-style-key)?  The [upstream commit](https://gitlab.com/fedora/docs/templates/fedora-docs-template/-/commit/bca134e1c05d93db7acecacdfefc4afabd8a1223) doesn't say why it was done.

Reply:
I've asked upstream. The docs are not built by this script however so this only impacts local testing. I'll update to not do that pending a response.